src/assets/sass/protocol/components/_button.scss

```diff
@@ -4,36 +4,153 @@
 
 @import '../includes/lib';
 
+/* -------------------------------------------------------------------------- */
+// Primary Button
+
 .mzp-c-button {
     @include border-box;
     @include text-cta;
-    transition: background-color 100ms, border-color 100ms, color 100ms;
+    transition: background-color 100ms, box-shadow 100ms, color 100ms, transform 100ms;
```

Comment:
Nit: use `transition` mixin. 

Alphabetising the CSS properties in here would also be nice for consistency (here and below).

Reply:
Alphabetising the CSS properties in in this file would also be nice for consistency (both here and below), although not a blocker.